nomos-core/chain-defs/src/da/blob/select.rs

```diff
@@ -31,6 +30,14 @@ impl<const SIZE: usize, B: DispersedBlobInfo> BlobSelect for FillSize<SIZE, B> {
         &self,
         certificates: I,
     ) -> impl Iterator<Item = Self::BlobId> + 'i {
-        utils::select::select_from_till_fill_size::<SIZE, Self::BlobId>(|c| c.size(), certificates)
+        #[expect(clippy::redundant_closure_for_method_calls)]
+        // TODO: Replace this redundant closure with `B::size` without triggering compiler errors
+        // about B not living long enough.
```

Comment:
What is the issue here. Calling `size`?

Reply:
Yes, replacing the closure with `B::size` gives lifetime issues. You now how to fix this?

Reply:
I can take a look. It kinda triggers something from when I implemented this piece. But I do not recall correctly. It doesn't really matter much.

Reply:
Sure, this is anyway a very minor details. We do ignore lints in few other places (especially the `too_many_lines` and the `cognitive_complexity` ones), so this would not be the only one we ignore and definitely not the "worst" one to ignore. But I'm curious to see how the solution would look like, as I tried adding a lifetime bound on `B` but it did not fix the compilation issue.

Reply:
FYI I will go ahead and merge this PR. Any changes to this bit here would be introduced in a new, small PR 😊